packages/namekit-react/src/components/Button.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import cc from "classcat";
+
+export interface ButtonProps
+  extends React.ButtonHTMLAttributes<HTMLButtonElement> {
+  asChild?: React.ReactElement;
+  className?: string;
+  children?: React.ReactNode;
+  variant?: "primary" | "secondary" | "ghost";
+  size?: "small" | "medium" | "large";
+}
+
+const buttonBaseClasses =
+  "nk-transition nk-text-base nk-rounded-lg nk-border nk-font-medium";
+
+const variantClasses = {
+  primary: "nk-bg-black nk-text-white nk-border-black hover:nk-bg-mine-shaft",
+  secondary:
+    "nk-bg-white nk-text-black nk-border-alto nk-shadow-[0_1px_2px_0_rgba(0,0,0,0.05)] hover:nk-bg-gray-50",
+  ghost: "nk-text-black nk-border-transparent hover:nk-bg-black/5",
+};
+
+const sizeClasses = {
+  small: "nk-py-1 nk-px-2 nk-text-sm",
+  medium: "nk-py-2 nk-px-4",
+  large: "nk-py-3 nk-px-6 nk-text-lg",
+};
+
+export const Button: React.FC<ButtonProps> = ({
```

Comment:
@notrab Great to see the updates here. My first impression is that the default styles here look pretty similar to what we have in Figma. Can you confirm if we've aligned with those 100%?

Reply:
Yeah, should be inline with Figma.